apps/provisioning_api/lib/Controller/UsersController.php

```diff
@@ -167,19 +167,21 @@ public function getUsers(string $search = '', ?int $limit = null, int $offset =
 	 * @param string $search Text to search for
 	 * @param int|null $limit Limit the amount of groups returned
 	 * @param int $offset Offset for searching for groups
+	 * @param string $orderBy Field to order the results with
+	 * @param string $sort ASC or DESC
```

Comment:
```suggestion
	 * @param string $sort asc or desc
```
Would use lowercase like our other API endpoints